scripts/base/freeroam.nut

```diff
@@ -31,12 +31,13 @@ function onPlayerJoin( player ) {
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
   switch ( cmd ) {
-    case "rules": {
-      Message( "No disturbing other's play. Kids only." );
-      break;
-    }
-    default: {
-      break;
-    }
+  case "rules":
```

Comment:
We should probably switch this to `if (cmd == "rules")` like in other scripts, or vice-versa. Basically just make it uniform :P

Can alternatively be done in a separate PR.